website/package.json

```diff
@@ -12,12 +12,18 @@
     "docusaurus": "docusaurus"
   },
   "devDependencies": {
+    "@docusaurus/module-type-aliases": "^2.0.0-beta.6",
     "@svgr/webpack": "^5.5.0",
-    "babel-plugin-styled-components": "^1.12.0"
+    "@tsconfig/docusaurus": "^1.0.4",
+    "@types/react": "^17.0.20",
+    "@types/react-helmet": "^6.1.2",
+    "@types/react-router-dom": "^5.1.8",
+    "babel-plugin-styled-components": "^1.12.0",
+    "typescript": "^4.4.2"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.0",
-    "@docusaurus/preset-classic": "2.0.0-beta.0",
+    "@docusaurus/core": "2.0.0-beta.6",
```

Comment:
could it be possible for us use `^2.0.0-beta.6` then we can use the latest docusaurus every time?

Reply:
If there is a destructive update that will crash our CI, I recommend not using the latest version before the official version is launched.

Reply:
got you, thanks @bzp2010 , then we need to keep an eye on the developments in the docusaurus community, we can follow the updates when it has important upgrades